model/src/main/java/io/a2a/spec/JSONParseError.java

```diff
@@ -1,11 +1,10 @@
 package io.a2a.spec;
 
-import static io.a2a.util.Utils.defaultIfNull;
-
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import static io.a2a.util.Utils.*;
```

Comment:
Wildcard import, also we like static imports to be on their own on top (like before this change)